arex-schedule-web-api/src/main/java/com/arextest/schedule/web/controller/ReplayPlanController.java

```diff
@@ -68,7 +67,15 @@ public CommonResponse reRunPlan(@RequestBody ReRunReplayPlanRequest request) {
         try {
             return planProduceService.reRunPlan(request.getPlanId());
         } catch (Throwable e) {
```

Comment:
It feels better to write catch with different exceptions

Reply:
fixed